docker-java-core/src/main/java/com/github/dockerjava/core/exec/StatsCmdExec.java

```diff
@@ -23,6 +25,10 @@ protected Void execute0(StatsCmd command, ResultCallback<Statistics> resultCallb
         WebTarget webTarget = getBaseResource().path("/containers/{id}/stats").resolveTemplate("id",
                 command.getContainerId());
 
+        if (isTrue(command.getNoStream())) {
```

Comment:
I believe we don't need `BooleanUtils` here and can simply use `Boolean.TRUE.equals(command.getNoStream())` (null safe)